src/js/components/Menu.js

```diff
@@ -2,17 +2,18 @@
 
 import React, { Component, PropTypes } from 'react';
 import ReactDOM from 'react-dom';
-import pick from 'lodash/object/pick';
-import keys from 'lodash/object/keys';
+import classnames from 'classnames';
+import isFunction from 'lodash/lang/isFunction';
```

Comment:
I agree. It's better to shorten the list of dependencies. We can do it with a try/catch block instead of  a conditional check.

I'll make changes if it's OK.